src/main/webapp/dev/js/common/instructorFeedbackResults.js

```diff
@@ -151,11 +151,14 @@ function expandOrCollapsePanels(expandCollapseButton, panels) {
     const STRING_COLLAPSE = 'Collapse';
     const expandCollapseStudentsButton = 'a[id^="collapse-panels-button-section-"]';
     const expandCollapseTeamButton = 'a[id^="collapse-panels-button-team-"]';
-    // {@code panels} is not defined when {@code expandCollapseButton} is collapse panels button. We
-    // need to define corresponding {@code targetPanels}.
-
     const isButtonInExpandMode = $(expandCollapseButton).html().trim().startsWith(STRING_EXPAND);
 
+    /**
+     *When collapsing (@code expandCollapseButton), all panels buttons under it
```

Comment:
I believe jsdoc uses curly braces. However, this can be just a usual comment (use `/*` instead of `/**`). Can remove the `code` annotations.